<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainWindowLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Virtual Machine - File transfer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hostToVMButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer a file from the host to the VM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vmToHostButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer a file from the VM to the host</string>
    </property>
   </widget>
   <widget class="QLabel" name="subLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Host &lt;--&gt; {vmName}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mountButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount the VM filesystem</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
